spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts controller', type: :request do
+  before(:example) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+                        posts_counter: 0)
+  end
+
+  it 'renders posts of a user' do
+    get "/users/#{@user.id}/posts"
+
+    expect(response).to have_http_status(:ok)
+
+    expect(response).to render_template(:index)
+
+    expect(response.body).to include('All posts for User #Tom')
+  end
```

Comment:
- [ ] I think you have two tests are failed please fix them to and try to solve the issues to be able to test your code in a perfect way as required ✨💯✅ because it is very important to write unit test so it is very recommended when you are searching for a new job the recruiters could see that you are be able to understand how it is important topic is!✅✅✅


### Kindly check the error messages ⬇⬇
![image](https://user-images.githubusercontent.com/58553711/214195228-80bd7fbc-4c57-45eb-afb1-a327c28eadd2.png)


- The First test
![image](https://user-images.githubusercontent.com/58553711/214195161-29dc9ad0-0166-4d30-907d-a5410bbe728e.png)


- The second test
![image](https://user-images.githubusercontent.com/58553711/214195196-f088c75d-8650-46d7-8a40-26e6691ae33d.png)